src/components/visit/VisitRegistrationReview.tsx

```diff
@@ -1,3 +1,4 @@
+import moment from 'moment';
```

Comment:
I honestly think that we should move away from using `moment` because `luxon` was introduced by STFC people as better library to work with when using dates and times and also it has better support. I see that the moment is not even part of our direct dependencies in the `package.json` but `luxon` is, so I guess there is all the functionality that we need. There are few other places that we should change to use luxon instead of moment but that could come with another PR.

Reply:
Yes, I think we should have abstraction util for time methods so that if we decide to switch either library or time format or have it controlled differently per site there is one central place to do it.